### ISSUE TITLE
Project without love anymore?

### ISSUE THREAD

**Issue by EThaNol2005** [PoE-TradeMacro/POE-TradeMacro#1007]
Hi, since leagues i got many times that new/old affixes dosn't supported by TradeMacro. 
as example :
![](https://puu.sh/DQab0/b26592e27e.png)
there is definitely a support for the affix on poe.trade. It's really a lot that is missing atm.

poeaffix has since month broken links, they are reported, but not hotfixed. 

I think TradeMacro is such a important tool for poe and his community. But there is no more Wip, i see only little fixes, so the tool isn't completely unable to use. 

If you have no time or don't play poe anymore, let us help you. The community is nice and active. I see a nice pull request that updates divcards and maps. This is a easy merch into your master. 

much love etha

**Comment by Eruyome** [comment]
Yes, I'm neglecting it a bit, but I do stuff aside from fixes, which is prototyping stuff like supporting the official trade site and making huge improvements to the tooltip.

Anyways, issues like that Tornado Shot enchantment are tricky. I'm scraping the wiki for all those enchantments and in general they are fine but here is what I get for TS:

```
{
"id": "EnchantmentTornadoShotNumOfSecondaryProjectiles1",
"mod type": "EnchantmentTornadoShotNumOfSecondaryProjectiles",
"stat text": "[[Tornado Shot]] fires an additional secondary Projectile",
"tag": "helmet"
},
{
"id": "EnchantmentTornadoShotNumOfSecondaryProjectiles2",
"mod type": "EnchantmentTornadoShotNumOfSecondaryProjectiles",
"stat text": "[[Tornado Shot]] fires an additional secondary Projectile",
"tag": "helmet"
}

```

while poe.trade uses
```
 Tornado Shot fires # additional secondary Projectiles
```

I can't hardcode, check and fix every enchantment.
